### PR TITLE
fix: add Dhruva Srinivas

### PR DESCRIPTION
**Pull request by carrotfarmer**
## Fixes Issue
- added my profile

## Changes proposed
- `CREATE public/data/carrotfarmer.json`

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/793"></a>

### CODE REVIEW

File: public/data/carrotfarmer.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Dhruva Srinivas",
+  "type": "personal",
+  "bio": "backend dev & linux/OSS enthusiast",
```
Comment (Panquesito7) [inline public/data/carrotfarmer.json:4]:
```suggestion
  "bio": "Backend developer and Linux/OSS enthusiast",
```

File: public/data/carrotfarmer.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "avatar": "https://github.com/carrotfarmer.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
```
Comment (Panquesito7) [inline public/data/carrotfarmer.json:9]:
```suggestion
      "url": "https://github.com/carrotfarmer",
```

**Review by Pradumnasaraf — APPROVED** [review]
LGTM 🚀

**Review by krishguptadev — APPROVED** [review]

**Comment by carrotfarmer** [comment]
👍🏻

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**